client/lib/plans/constants.js

```diff
@@ -705,7 +803,89 @@ export const FEATURES_LIST = {
 		getDescription: () => i18n.translate(
 			'Custom polls, surveys, ratings, and quizzes for the ultimate in customer and reader engagement.'
 		)
-	}
+	},
+	[ FEATURE_CORE_JETPACK ]: {
+		getSlug: () => FEATURE_CORE_JETPACK,
+		getTitle: () => i18n.translate( 'Core Jetpack Services' ),
+		getDescription: () => i18n.translate(
+			'Stats, customization, and promotion tools'
```

Comment:
Alternate string suggestion: 
* `translate( 'Traffic and promotion tools' )`&mdash; translations: **18**. 

[PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/try/jetpack-plans-tab)